fix: replaced create campaign image with 5.2 mautic app version image

beneyalraj opened the pull request:
- I have updated docs/campaigns/images/new-campaign.png. reviewed .rst file as well
- updated image is from 5.2 version of mautic app
- updated image format is png 
- used light mode to take screenshots
- tested and previewed changes locally, renders fine

<!-- PLEASE WRITE ABOVE THIS COMMENT. -->

<!--

Clearly describe what changes you have made in this PR, where our maintainers or reviewers should look at, and how to test the changes.

If the request is not complete but you want feedback or have quetions, you can select the "Draft Pull Request" option from the dropdown menu when creating the PR, then ask your questions or write your feedback in the comment.

-->

Closes #376


<!-- PLEASE WRITE ABOVE THIS COMMENT. -->

<!--

If your PR is related to a current issue, please link to that issue number. Type the keyword "Closes" followed by a hashtag (#) symbol and the issue number that you can find right after the issue title. Don't add anything else, such as period, comma, etc, to this. For example:

❌ Closes: #123.

✅ Closes #123

Doing so will automatically close the issue when one of our maintainers merges your PR.

-->

## Screenshots or screen recordings
BEFORE CHANGES -
<img width="1358" height="823" alt="Screenshot 2025-10-03 at 6 15 14 PM" src="https://github.com/user-attachments/assets/7529277d-f6de-46e5-8f8e-05316e366cd4" />

AFTER CHANGES - 
<img width="1351" height="854" alt="Screenshot 2025-10-03 at 9 48 07 PM" src="https://github.com/user-attachments/assets/f2c1fa95-435c-45a2-ae9f-b8b4c3966283" />


<!-- 

Provide screenshots or screen recordings before and after the changes, if it's a UI changes. You can simply drag and drop your files above this comment.

-->

beneyalraj committed:
fix: replaced create campaign 5.2 version image

adiati98 reviewed (changes requested):
Thanks for the PR, @beneyalraj. ✨ 

Can you make sure that this is the UI of version 5.2?
I think the "Save & Close" and "Cancel" buttons have white background in that version.

beneyalraj commented:
yes @adiati98 

- This is the 5.2 version of the app
- I have cross checked it with the email builder template page,  5.2 version has no template other than blank one
- Also, I have only cloned only the 5.2 branch from mautic repo which is available in my local machine
- i have attached the below image from the 5.2 version running currently

<img width="1351" height="814" alt="Screenshot 2025-10-03 at 9 10 14 PM" src="https://github.com/user-attachments/assets/f17b2bdd-7f54-4515-b4fd-5084986aaa7e" />

beneyalraj commented:
@adiati98 please wait, i have attached the wrong screenshot

was supposed to add this, let me change it quickly

<img width="1191" height="533" alt="Screenshot 2025-10-03 at 9 19 36 PM" src="https://github.com/user-attachments/assets/203940a4-ba3f-43ab-bd48-676eb00a83ed" />

adiati98 commented:
So it's different with the image that you've committed. That one has a black "Cancel" button.

beneyalraj committed:
docs(5.2): replaced the image, now the buttons are in white as expected

beneyalraj committed:
fixed some clash issue

beneyalraj commented:
Correct! 

I have taken this screenshot 2 days earlier when I was on 7.x version which you have identified since then I'm on 5.2
This wont be happening again as i don't have any more screenshots from 7.x version

Good catch @adiati98 🫡

adiati98 reviewed (commented):
A question here. Why you removed this image: `docs/builders/images/editorfonts.jpg`?
This image is not part of the [Creating Campaigns](https://docs.mautic.org/en/5.2/campaigns/creating_campaigns.html) page.

beneyalraj commented:
because in the previous PR that i have raised..

- I have changed the docs/builders/images in my previous PR, there were 2 images
- This .JPG image that you are referring to is the old one which was there on the doc before(task was to remove this and update the new 5.2 version in .png format)
- currently after my PR both the images are .PNG and i have removed it in the previous PR itself 
- but i did not commit this change (even now this has no effect on the repo cause there is no docs/builders/images/editorfonts.jpg to begin with )
- Now, I have committed this because i had some conflicts with the base branch 
- should have committed in the last PR, This wont do anything to the repo

let me know your thoughts on this. how would you handle such situations

adiati98 commented:
What merge conflicts did you get? In my understanding, it shouldn't give any merge conflicts. 

Did you create this branch from your last branch or start fresh from branch `5.2`?

beneyalraj commented:
<img width="1427" height="441" alt="Screenshot 2025-10-04 at 6 24 18 PM" src="https://github.com/user-attachments/assets/e7a8350c-025a-480a-881c-2b57582b5a63" />

- i did it from 5.2. but before that i pulled from the origin
- i had issues when i tried to push
- if my assumption is correct, there is still docs/builders/images/editorfonts.jpg in the offical mautic base repo.
- because i did not commit the removal change in my last PR
- this PR should remove the .jpg file and update the create campaign image as well

I am still getting used to Git, i'm sorry if it is causing you any problems 
thankful for the learning and guidance @adiati98

beneyalraj commented:
<img width="1428" height="564" alt="Screenshot 2025-10-04 at 6 33 55 PM" src="https://github.com/user-attachments/assets/9ce4aad7-6882-4bb8-8015-68988a423390" />

- this is the mautic 5.2 repo
- there is 3 images, .jpg one is not needed
- this PR should remove that image

adiati98 commented:
My concern here is because force push is used when I believe there's no need to do that in this case. My assumption here you used it was because there were conflicts?

Sometimes, there are cases when you need to do force push. But always with caution because it can mess up the remote base branch. 

Do you mind creating a new PR? This time, please: 

- Make sure you create a new branch directly from the branch that you need to work on. In this case, create a new branch from branch `5.2` --> always do this for any project.
- You can include the jpg removal, but please mention it in your PR
- When you create a PR, please don't delete the PR templates. I see that you've removed the titles in your PRs.

beneyalraj commented:
sure @adiati98, let me close this PR and raise a new PR which includes 

- removal of .jpg image in docs/builders/images
- updates of create campaign images

I will make sure to go through the PR template and keep the titles as well, or will include the titles of the issues it fixes
will mention what the PR is going to change clearly in the description.
